src-react/containers/path.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux'
 
+import "../styles/styles.scss";
```

Comment:
Fix eslint on another PR, 
and I recommend use `resolve.modulesDirectories` to solve all the problem about require() relative path hell.